platform/fio/transaction.go

```diff
@@ -2,8 +2,78 @@ package fio
 
 import (
 	"github.com/trustwallet/blockatlas/pkg/blockatlas"
+	"github.com/trustwallet/blockatlas/pkg/errors"
+	"encoding/json"
+	"strconv"
+	"strings"
+	"time"
 )
 
 func (p *Platform) GetTxsByAddress(address string) (page blockatlas.TxPage, err error) {
-	return page, err
+	account := address // TODO this should be actor
```

Comment:
// TODO this should be actor
What do you mean?

Reply:
The address (fio public key) has to be converted to 'actor', I'm working on this now.

Reply:
Done now.